contracts/GateSeal.vy

```diff
@@ -177,4 +226,4 @@ def _is_expired() -> bool:
 
 @internal
 def _expire_immediately():
-    self.expiry_timestamp = 0
\ No newline at end of file
+    self.expiry_timestamp = block.timestamp
```

Comment:
requires an additional change:
```vyper
@internal
@view
def _is_expired() -> bool:
    return block.timestamp >= self.expiry_timestamp
```
(`>=` instead of `>`)